Formula/kubetail.rb

```diff
@@ -0,0 +1,24 @@
+class Kubetail < Formula
+  desc "Bash script to tail Kubernetes logs from multiple pods at the same time"
+  homepage "https://github.com/johanhaleby/kubetail"
+  url "https://github.com/johanhaleby/kubetail/archive/1.6.13.tar.gz"
+  sha256 "a4aeea1cffede44d5e8b030b6ead3ffe25fd3c1e7c5b7c82905df65cceac1254"
+  license "Apache-2.0"
+  head "https://github.com/johanhaleby/kubetail.git", branch: "master"
+
+  depends_on "kubernetes-cli" => :test
+
+  def install
+    bin.install "kubetail"
+
+    bash_completion.install "completion/kubetail.bash" => "kubetail"
+    zsh_completion.install "completion/kubetail.zsh" => "_kubetail"
+    fish_completion.install "completion/kubetail.fish"
+  end
+
+  test do
+    assert_match "server localhost:8080 was refused",
+      shell_output("#{bin}/kubetail brew-pod-v1 2>&1", 1)
```

Comment:
Can we tell it to use a specific port? This is a pretty common port to be in use.